reset.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```

Comment:
Please add a security to make sure this can NOT be run on production instance ⚠️ @nicolas-heigvd